src/grt/src/Grid.cpp

```diff
@@ -187,14 +187,14 @@ int Grid::computeTileReduce(const odb::Rect& obs,
 
 odb::Point Grid::getMiddle()
 {
-  return odb::Point((lower_left_x_ + (upper_right_x_ - lower_left_x_) / 2.0),
-                    (lower_left_y_ + (upper_right_y_ - lower_left_y_) / 2.0));
+  return odb::Point((die_area_.xMin() + (die_area_.xMax() - die_area_.xMin()) / 2.0),
+                    (die_area_.yMin() + (die_area_.yMax() - die_area_.yMin()) / 2.0));
 }
 
 odb::Rect Grid::getGridArea() const
 {
   return odb::Rect(
-      lower_left_x_, lower_left_y_, upper_right_x_, upper_right_y_);
+      die_area_.xMin(), die_area_.yMin(), die_area_.xMax(), die_area_.yMax());
```

Comment:
Could just be
```
return die_area_;
```
A copy is already made based on the return type. If it could be `const odb::Rect&` that would be even better.